test-app/src/main/java/net/rubygrapefruit/platform/test/Main.java

```diff
@@ -415,7 +415,7 @@ private static FileWatcher createWatcher(String path, FileWatcherCallback callba
                 .startWatcher(callback);
         } else if (Platform.current().isWindows()) {
             watcher = Native.get(WindowsFileEventFunctions.class)
-                .startWatcher(callback);
+                .startWatcher(64 * 1024, callback);
```

Comment:
Why are we using 64k here and 16k in the tests?

Reply:
Any of these numbers are guesswork. I used 16K for the tests as a kind of "almost too little" number, so we'd notice if anything started to fail soon for our tests that should have very few events. 4x that sounds like reasonable to avoid unnecessary overflows in the test app if there are lots of events.